src/sources/redis/mod.rs

```diff
@@ -187,14 +191,29 @@ impl SourceConfig for RedisSourceConfig {
                     self.redis_key.clone(),
                     decoder,
                     cx,
+                    log_namespace,
                 )
                 .await
             }
         }
     }
 
-    fn outputs(&self, _global_log_namespace: LogNamespace) -> Vec<Output> {
-        vec![Output::default(self.decoding.output_type())]
+    fn outputs(&self, global_log_namespace: LogNamespace) -> Vec<Output> {
+        let log_namespace = global_log_namespace.merge(self.log_namespace);
+
+        let schema_definition = self
+            .decoding
+            .schema_definition(log_namespace)
+            .with_standard_vector_source_metadata()
+            .with_source_metadata(
+                "redis",
```

Comment:
seeing the raw string `"redis"` a lot in this file. :thinking:  I wonder if this sink is setup to get the sink name from the `configurable` stuff.
I do believe @spencergilbert shall know better than I.

If not, we could have a const at the top of the file and use it instead of the raw string literals:

```Rust
const SINK_NAME: &str = "redis";
```

Reply:
You should be able to use `RedisSourceConfig::NAME`, which is generated by the `configurable` macro magic. (This didn't exist when I created the `demo_logs` example)

Reply:
Yep, depending on where you're calling from... I think this could also be a `self.get_component_name()` - that should work in this location since `RedisSourceConfig` == `self`.

Reply:
Based on the feedback on my PR, sounds like we'd prefer to be consistent on `Self::NAME` or `RedisSourceConfig::NAME`